### PR TITLE
Update version for default packages

### PR DESCRIPTION
**Pull request by victoray**
**Contributing to Twilio**

> All third-party contributors acknowledge that any contributions they provide will be made under the same open-source license that the open-source project is provided under.

- [x] I acknowledge that all my contributions will be made under the project's license.

### CODE REVIEW

File: packages/runtime-handler/package.json [hunk 1]
```diff
@@ -44,6 +44,7 @@
     "@types/cookie-parser": "^1.4.2",
     "@types/debug": "^4.1.4",
     "@types/express-useragent": "^0.2.21",
+    "@types/express-serve-static-core": "4.17.33",
```
Comment (victoray) [inline packages/runtime-handler/package.json:47]: an update to this package broke tests, so pinning to this version

File: .github/workflows/nodejs.yml [hunk 2]
```diff
@@ -27,6 +27,8 @@ jobs:
       - name: Update npm to 8.x
         run: |
           npm i -g npm@8
+          npm --version
+          node --version
```
Comment (AndreLars) [inline .github/workflows/nodejs.yml]: do we need this here or is it just debug?

File: tsconfig.base.json [hunk 3]
```diff
@@ -2,7 +2,7 @@
   "compilerOptions": {
     /* Basic Options */
     "incremental": false /* Enable incremental compilation */,
-    "target": "es2015" /* Specify ECMAScript target version: 'ES3' (default), 'ES5', 'ES2015', 'ES2016', 'ES2017', 'ES2018', 'ES2019' or 'ESNEXT'. */,
+    "target": "es2018" /* Specify ECMAScript target version: 'ES3' (default), 'ES5', 'ES2015', 'ES2016', 'ES2017', 'ES2018', 'ES2019' or 'ESNEXT'. */,
     "module": "commonjs" /* Specify module code generation: 'none', 'commonjs', 'amd', 'system', 'umd', 'es2015', or 'ESNext'. */,
     "lib": [
       "es2017",
```
Comment (AndreLars) [inline tsconfig.base.json]: should we move this to es2018?

**Review by AndreLars — APPROVED** [review]

File: .github/workflows/nodejs.yml (hunk 2 above)
Comment (victoray) [reply .github/workflows/nodejs.yml]: debug, will remove

File: tsconfig.base.json (hunk 3 above)
Comment (victoray) [reply tsconfig.base.json]: i don't see any issues with that

**Pull request merged**